File: 백준/Bronze/33541. 2025는 무엇이 특별할까？/2025는 무엇이 특별할까？.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.lang.Math;
public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n1,n2,st = Integer.parseInt(br.readLine());
        while(true) {
            n1=st/100;
            n2=st%100+1;
            st+=1;
            if(st>9999) {
                System.out.print(-1);
                break;
            }
            if(st==Math.pow(n1+n2,2)) {
                System.out.print(st);
                break;
            }
        }
    }
}

File: 백준/Bronze/33541. 2025는 무엇이 특별할까？/2025는 무엇이 특별할까？.c
#include <math.h>
#include <stdbool.h>
#include <stdio.h>
int main(void) {
    int st, n1, n2;
    scanf("%d", &st);
    while (true) {
        n1 = st / 100, n2 = st % 100 + 1;
        st += 1;
        if (st > 9999) {
            printf("%d", -1);
            break;
        }
        if (st == pow(n1 + n2, 2)) {
            printf("%d", st);
            break;
        }
    }
    return 0;
}